src/robotide/ui/tagdialogs.py

```diff
@@ -161,8 +170,7 @@ def _search_for_tags(self):
         isreversed = (self.sort_state[1] != 1)
         self.total_test_cases = len(self._test_cases)
         self._results = sorted(self._unique_tags.items(),
-                               key=lambda x: str(x[0]).lower,
-                               reverse=isreversed)
+                               key=lambda x: str(x[0]).lower, reverse=isreversed)
```

Comment:
1 and 2. It was an unnoticed change when squashing, I also like better as it was.